backend/yonder/views.py

```diff
@@ -169,3 +173,33 @@ def put(self, request, *args, **kwargs):
     @swagger_auto_schema(tags=['comments'])
     def delete(self, request, *args, **kwargs):
         return self.destroy(request, *args, **kwargs)
+
+class inbox(generics.ListCreateAPIView):
+
+    @swagger_auto_schema(tags=['inbox'])
+    def get(self, request, *args, **kwargs):
+        # if not request.user.is_authenticated:
+            # return Response(status=status.HTTP_401_UNAUTHORIZED)
+
+        inbox = get_object_or_404(Inbox.objects.all(), author_id=kwargs["author_id"])
```

Comment:
A bit of a performance issue here, this should be `get_object_or_404(Inbox, ....)` instead.
`get_object_or_404(Inbox.objects.all(), ....)` retrieves all Inboxes first then gets/filters for the correct one, instead of just allowing django to filter/get specifically by `author_id` and just return that.